ekip/redemption/views.py

```diff
@@ -48,15 +48,16 @@ def csv_redemption(request):
     response = HttpResponse(content_type='text/csv')
     response['Content-Disposition'] = 'attachment; filename="exchanges.csv"'
 
-    start_date = request.GET.get('start', '20150901')
```

Comment:
Adding a test for this would also make sense I think. 
Also, I wonder if the default should be the beginning of the year, or the beginning of the month.